src/frontend/sharedComponents/Images/TrulyContainedImage.tsx

```diff
@@ -12,32 +15,28 @@ export function TrulyContainedImage({
   onLoad,
   style,
   ...baseImageProps
-}: Omit<
-  ImageProps,
-  // Resize mode has no effect.
-  | 'resizeMode'
-  // Dimensions are dictated by the container that lives in the consuming component.
-  | 'height'
-  | 'width'
->) {
+}: Omit<ExpoImageProps, 'contentFit'>) {
   const [aspectRatio, setAspectRatio] = useState<number | undefined>(undefined);
 
   return (
-    <Image
+    <ExpoImage
       {...baseImageProps}
-      resizeMode="contain"
+      contentFit="contain"
       style={[
         style,
         // There seems to be a bug related to images that have non-network sources (e.g. `file://...`)
         // where the loading the image won't even be attempted unless the image has a known height prior.
         // We trigger the load attempt by initially setting the height to a non-zero value and then removing it once it loads.
-        {aspectRatio, height: aspectRatio ? undefined : 1},
+        // Note: with React Native's image, the initial height can be set to 1 but with Expo's Image, it needs to be a larger value or else
+        // there's some layout glitchiness each time it loads...
+        {
+          aspectRatio,
+          height: aspectRatio ? undefined : 50,
+        },
```

Comment:
a rather gnarly difference. haven't had the energy to look into the issues to see if there's anything about workarounds or updates.